apps/renterd/components/Files/checks/useNotEnoughContracts.tsx

```diff
@@ -3,7 +3,8 @@ import { useRedundancySettings } from '../../../hooks/useRedundancySettings'
 
 export function useNotEnoughContracts() {
   const redundancy = useRedundancySettings()
-  const { datasetCount, isLoading: isContractsLoading } = useContracts()
+  const { dataset, isLoading: isContractsLoading } = useContracts()
```

Comment:
Im unsure if it's better to it this way or change datasetCount in useContracts() to use the unfiltered dataset, and another variable for pagination called datasetFiltererdCount and update that code as well.

Reply:
Hey, thanks so much for the PR! I think it would be clearest to have `datasetCount` and `datasetFilteredCount` (and `pageCount`).

Reply:
Alright, I've given that a try as well!